ansible/group_vars/store-db.yml

```diff
@@ -55,6 +55,8 @@ postgres_system_setting_stage:
     ## Autovacuum
     autovacuum: 'off'
 
+postgres_share_memory: '{{ ansible_memtotal_mb / 16 | int }}m'
```

Comment:
If possible, I think is interesting to add a comment nearby explaining why dividing by 16 so that we remember it in the future